sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfig.java

```diff
@@ -70,4 +77,18 @@ public Double getTargetThroughputThreshold() {
     public boolean isDefault() {
         return this.isDefault;
     }
+
+
+    /**
+     * Get whether request is allowed to continue on original request flow if throughput control controller failed on initialization.
+     *
+     * By default, it is false.
+     * If it is true, requests will continue on original request flow if throughput control controller failed on initialization.
+     *
+     * @return {@code true} request will continue on original request flow if throughput control controller failed on initialization. {@code false} otherwise.
+     */
+    @Beta(value = Beta.SinceVersion.V4_28_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public boolean isContinueOnInitError() {
```

Comment:
there is no immediate need for now, just using the same pattern as other properties

Reply:
okay, if others are also exposed, we should think about whether they should be exposed or not as well.

Reply:
IMO nor worth breaking changes (even in beta api). I agree that we might not have needed to expose them - but they also don't harm.

Reply:
Discussed offline.
The best practice is that we expose when there is some real needs. But since we have exposed previously, for consistency, we are going to expose this one as well.